weldx/welding/utility.py

```diff
@@ -0,0 +1,41 @@
+"""Collection of welding utilities."""
+import pint
+import numpy as np
+
+from weldx.constants import WELDX_UNIT_REGISTRY
+from weldx.welding.groove.iso_9692_1 import IsoBaseGroove
+
+
+@WELDX_UNIT_REGISTRY.check(None, "[length]", "[length]/[time]", "[length]")
+# TODO: extend the ckeck decorator to check on output! # ret='[length]/[time]')
+def compute_welding_speed(
+    groove: IsoBaseGroove,
+    seam_length: pint.Quantity,
+    wire_feed: pint.Quantity,
+    wire_diameter: pint.Quantity,
+):
+    """Computes how fast the torch has to be moved to fill the gap of the groove.
+
+    Parameters
+    ----------
+    groove: IsoBaseGroove
```

Comment:
```suggestion
    groove
```
I suggest we omit the type info from the docstrings in cases where they match the function type hints (like here)
The missing type info will be inherited from type hints in the documentation.
https://weldx--248.org.readthedocs.build/en/248/_autosummary/weldx.welding.utility.compute_welding_speed.html#weldx.welding.utility.compute_welding_speed

This way it is easier to keep everything synced up and the intention is clear when a specific type info is listed in the docstring (showing a "clearer" info than the type hints like for this case)
I like the `pint.Quantity["length"]` approach btw